BUGS FIXED AND NEW LIB ADDED

Ansh101112 opened the pull request:
FIXED SOME BUGS IN BACKEND AND ADD NEW LIBRARY TOO. 

@usha-madithati @Durgesh-Vaigandla

usha-madithati commented:
Sir @sanjay-kv .Firstly regarding merging that pr I have checked while merging that pr the code was all correct sir  and even the checks of deployment were passed. I haven't merged any pr with conflicts. But if I have merged that pr because it wasn't making any conflicts with the inner code and even the deployment of website wasn't affected sir so I thought its ok to merge that . Regarding that merging direct prs before even raising issue ...I would like to tell you that this pr was merged 1 month back . After your instructions sir I haven't done that . One more thing  I would like to tell you that many contributors are raising the issues and even working on those issues and they were raising prs for them before we even assign them to work . 
I have reviewed each and every code file of our website sir . It's just that I haven't clicked the viewed check box sir . But I have checked every code file sir .
Thank you sir.

Ansh101112 commented:
> ![image](https://github.com/usha-madithati/esmart.github.io/assets/30715153/e00ff3f1-9dc1-47a4-b616-7359c1ff639a)
> @usha-madithati 
> 
> You merged this PR with conflicts and without reviewing. Also I noticed none of the code you have reviewed on github for other contributors as well. 
> 
> @Ansh101112 you gotta explain whats the bugs and you again did direct PR without raising issues. 

Sorry sir it's a previous month pr where I have fixed some bugs and added a new library for showing alerts or something. And about the conflict in pr is caused due to the library changing which got fixed in the next prs as well. As discussed with PA @usha-madithati

sanjay-kv commented on src/pages/QR.js:
```diff
@@ -4,23 +4,44 @@ import Navbar from "../components/Navbar";
 import Footer from "../components/Footer";
 import AddProductForm from "./AddProductForm";
 import { Oval } from "react-loader-spinner";
+<<<<<<< HEAD
```
There is merge conflicts in the, this create unable to execute in main branch. you creating multiple bug issues to solve this.

Ansh101112 replied:
No sir why I create more bug to solve this one. As it's the that time or when there is addition of backend and changing of server from one deployment service to other and adding a new lib or removing one which creates fixing that causing bugs.
